examples/anexia-machinedeployment.yaml

```diff
@@ -48,6 +48,30 @@ spec:
               - size: 60
                 performanceType: ENT6
 
+            # each entry in this array will be created as a network interface
+            # of each Machine, connected to the given VLAN
+            networks:
+              - vlan: "<< ANEXIA_VLAN_ID >>"
+
+                # If prefixes are given, we reserve an IP address for each of
+                # them - if you give one IPv4 and one IPv6 prefix, your
+                # Machines will have dual-stack connectivity
+                #
+                # As an compatibility-aid for the old cloudProviderSpec.vlanID,
+                # which reserved an IP for the configured VLAN, you can also
+                # have an entry "" (empty string) to get the same behavior -
+                # but this is not recommended.
+                #
+                # Not configuring any prefix might be useful if you want to
+                # configure IP addresses on this interface via other means,
+                # e.g. a Layer2 load balancer.
+                #
+                # Each MachineDeployment needs at least one Network with at
+                # least one Prefix, so each Machine needs to get one reserved
+                # IP address.
```

Comment:
> needs to get one reserved IP address

That sentence implicates manual intervention from the operators. Wouldn't

> gets one IP address reserved

be a better description of the behavior?

Reply:
Changed the comment a bit, yet another version